apps/communications/contacts/test/marionette/import_sdcard_test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var Contacts = require('./lib/contacts');
+var assert = require('assert');
+
+marionette('Import from SDCARD', function() {
+  var profile = Contacts.config;
+  // Enable device storage
+  var prefs = ['device.storage.enabled',
+        'device.storage.testing',
+        'device.storage.prompt.testing'];
+  prefs.forEach(pref => {
+    profile[pref] = true;
+  });
+  var client = marionette.client({
+    profile: profile,
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var actions, subject, selectors;
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+
+    // Add the default vcard file
+    client.fileManager.add({
+      type: 'text',
+      filePath: 'apps/communications/contacts/test/marionette/data/vcard_4.vcf'
+    });
+
+    subject = new Contacts(client);
+    actions = client.loader.getActions();
+    subject.actions = actions;
+    subject.launch();
+    selectors = Contacts.Selectors;
+  });
+
+  test('> one contact in one file', function() {
+    // Go to settings
+    client.helper.waitForElement(selectors.settingsButton).click();
```

Comment:
I understand, and agree, but the support for gij iin contacts is really old, actually the very first one apps. I would consider to move to Page Object Pattern in a follow up (if we do it), but now prefer to have the tests up and running.